packages/changeset-bot/index.js

```diff
@@ -0,0 +1,104 @@
+module.exports = app => {
+  app.log('Yay, the app was loaded!');
+
+  // Get an express router to expose new HTTP endpoints
+  // Healthcheck
+  const router = app.route('/');
+  router.get('/healthcheck', (req, res) => {
+    res.send('OK');
+  });
+
+  app.on('pull_request.opened', async context => {
+    const params = context.issue();
+
+    const getFiles = context.github.pullRequests
+      .listFiles(params)
+      .then(files => {
+        const changesetFiles = files.data.filter(file =>
```

Comment:
This has false positives, as deleting an existing changeset will trigger a 'has changeset' display.